public/js/plantPlacement.js

```diff
@@ -1,86 +1,188 @@
+        var circle = draw.circle(40, 40).attr({ fill: 'grey' , opacity: 0.3, cy: i*75 + 30, cx: n*90+30}).data('key', {n,i})
+        if (SelectorNum) {
+          // if (pselect.length < 1) {
+            var circle = draw.circle(40, 40).attr({
+              fill: selectors[SelectorNum].color,
+              opacity: 0.9,
+              cy: i * 75 + 30,
+              cx: n * 90 + 30
+            }).data('key', {
+              x,
+              y
+            })
+            pselect.push(SelectorNum["plant"])
+        } else {
+      var circle = draw.circle(40, 40).attr({
+            fill: 'grey',
+            opacity: 1,
+            cy: i * 75 + 30,
+            cx: n * 90 + 30
+          }).data('key', {
+            x: n,
+            y: i
+          })
+       }
+        circle.click(function () {
+          this.fill({
+            color: color,
+            opacity: 1
+          })
+          var a = this.data('key')
+          var b = this.data('selectionNumber')
+          console.log(this.data());
+          plantMatrix.push({
+          selection: selectionNumber,
           location: a
-        });
-        //plantMatrix.push([color, a]);
-        console.log(color, this.data('key'));
+          });
+          //plantMatrix.push([color, a]);
+          console.log(color, this.data('key'));
         console.log(JSON.stringify(plantMatrix));
         document.getElementById("individualPlants").value = JSON.stringify(plantMatrix);
+        })
+      }
+    }
+  }
+
+
+  function makeSelector(index, selectorColor) {
+    var draw = SVG().addTo(`#selector${index}`).size(40, 80)
+    var circle = draw.circle(40, 40).attr({
+      fill: selectorColor,
+      cx: 20,
+      cy: 20
+    })
+    circle.on('click', function () {
+      clearSelected()
+      select(selectors[index])
+    });
 
+    selectors[index] = {
+      circle,
+      color: selectorColor,
+      selected: false,
+      selectionNumber: index,
+    }
 
+    function select(s) {
+      s.selected = true
+      s.circle.stroke({
+        color: 'black',
+        opacity: 0.6,
+        width: 2,
       })
+
+      color = s.color
+      selectionNumber = s.selectionNumber
+    }
+
+    function clearSelected() {
+      selectors.forEach(s => {
+        s.selected = false
+        s.circle.stroke({
+          width: 0,
+        })
+      });
     }
   }
-}
 
-$(document).ready(() => {
-  var article = document.querySelector('#plants');
-  console.log(article);
-  drawMod('rectangle', 'etuhc4545');
+
+  function clicker(ss, sel) {
+    ss.stroke({
+      color: 'black',
+      opacity: 0.6,
+      width: 2
+    })
+    color = ss.attr('fill')
+  }
+
+
+  $(document).ready(function() {
+    makeSelector(1, 'orange');
+    makeSelector(2, 'red');
+    makeSelector(3, 'yellow');
+    makeSelector(4, 'green');
+    makeSelector(5, 'blue');
+    defineUniquePlants();
+    drawMod('rectangle');
+    console.log(uniquePlants);
+    function individualPlants() {
```

Comment:
where's this fn used? it's a bit misleading to have it inside the document.ready since it's not used here